model/distro/distro_test.go

```diff
@@ -638,3 +638,36 @@ func TestGetAuthorizedKeysFile(t *testing.T) {
 		assert.Equal(t, expected, d.GetAuthorizedKeysFile())
 	})
 }
+
+func TestGetDistrosForImage(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	testConfig := testutil.TestConfig()
+	assert := assert.New(t)
+	session, _, err := db.GetGlobalSessionFactory().GetSession()
+	assert.NoError(err)
+	require.NotNil(t, session)
+	defer session.Close()
+	require.NoError(t, session.DB(testConfig.Database.DB).DropDatabase())
```

Comment:
I think we don't actually need all of this stuff; you can just have
```suggestion
	ctx, cancel := context.WithCancel(context.Background())
	defer cancel()

	assert.NoError(t, db.ClearCollections(Collection))
```
`db.ClearCollections` clears the collection in case there is some leftover data from running another test.